scripts/check-tests-number.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+import sys
+import xml.etree.ElementTree as ET
+
+if len(sys.argv) < 3:
+    print("Checks a JUnit XML report contains a minimum number of tests")
+    print("Usage: %s PATH_TO_REPORT MINIMUM_NUMBER" % sys.argv[0])
+    print("Example: %s build/jest-junit/unit-tests.xml 400" % sys.argv[0])
+    sys.exit(1)
+
+test_count = int(ET.parse(sys.argv[1]).getroot().attrib['tests']);
+print('%s tests have been run' % test_count)
```

Comment:
Sounds very specific, also introduces a vulnerability towards the testing framework logic e.g. if some tests are moved in the wrong folder they won't be found and run, but the step would be ok with it due to the minimum number shrinking at the same time. This goes for a [Literal Value](http://xunitpatterns.com/Literal%20Value.html), but with minimum rather than equality check to avoid having to update it often, as for coverage percentages.